config/routes.rb

```diff
@@ -14,6 +14,7 @@
       post 'domain_request'
       post 'delete_domain'
       post 'delete_all_data'
+      put 'appearance' => 'authors#update_appearance'
```

Comment:
I don't think this should be it's own endpoint. Should just be part of Authors#Update.

Reply:
@mobitar I tried with that approach at first and we have two options there:
- We always send all the author attributes (on the "General" section we send the "Appearance" data and viceversa)
- On the `update` method we check the params and if they're `nil` we don't update them
Which one do you think is the best?

Reply:
I think sending all attributes makes the most sense. I would also research into the most idiomatic way to do this with Rails. I believe it is the first option.